update-imports.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Update imports that reference the UI components
+find apps/back-office/src -type f -name "*.tsx" -o -name "*.ts" | xargs sed -i '' -E 's|from "../ui/([^"]+)"|from "@mcw/ui"|g'
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)